.github/workflows/e2e-tests-with-operator.yml

```diff
@@ -0,0 +1,165 @@
+name: End to End Tests with Operator #non-appsignals testing
+on:
+  workflow_call:
+    inputs:
+      aws-region:
+        required: true
+        type: string
+      image_tag:
+        required: true
+        type: string
+      image_name:
+        required: true
+        type: string
+      test_ref:
+        required: true
+        type: string
+      caller-workflow-name:
+        required: true
+        type: string
+
```

Comment:
Maybe we will need to add concurrency since we are running at main-build and nightly build.

Reply:
Done. Thanks Vasi.